modules/chronograf-security-group-rules/main.tf

```diff
@@ -0,0 +1,23 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ATTACH SECURITY GROUP RULE TO ALLOW INCOMING CONNECTIONS ON CHRONGRAF'S LISTENING PORT
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_security_group_rule" "http_port_cidr_blocks" {
+  count             = "${signum(length(var.http_port_cidr_blocks))}"
```

Comment:
Will these cidr_blocks always be static?

Reply:
CIDR blocks are usually specified in the form `["0.0.0.0/0"]` and are passed as-is to `cidr_blocks` argument of `aws_security_group_rule`. So they don't require an additional `*_num` variable that specifies their length